README.md

```diff
@@ -87,8 +87,22 @@ this.AmCharts.updateChart(this.chart, () => {
 
 You can see some examples in the `examples` directory.
 
+## Development
```

Comment:
I'm going to publish an npm package really soon, so I don't think these directions are necessary.